lib/smart_answer/calculators/pay_leave_for_parents_calculator.rb

```diff
@@ -16,6 +16,7 @@ class PayLeaveForParentsCalculator
     attr_accessor :partner_still_working_on_continuity_end_date
     attr_accessor :partner_earned_more_than_lower_earnings_limit
     attr_accessor :partner_worked_at_least_26_weeks
+    attr_accessor :partner_earned_at_least_390
```

Comment:
Is this used anywhere? It seems odd that no other code in the flow or template has had to change.